main.c

```diff
@@ -540,7 +553,16 @@ int main(int argc,char **argv) {
             }
 
             struct in_addr toAddress;
-            if (rcv_inaddr.s_addr == INADDR_BROADCAST
+            if (target_addr_override) {
+                // user instructed us to override the target IP address
+                if (target_addr_override == INADDR_BROADCAST) {
+                    // rewrite to new interface broadcast addr if user specified 255.255.255.0
```

Comment:
I think the `255.255.255.0` is a typo?  Shouldn't it be `.255` at the end?

Reply:
D'oh.  Yes.  Because I am dumb I originally used the input value 255.255.255.0 for the special value and then realized later that makes no sense and changed it to 255.255.255.255.  But then never updated the comment :-) 

Should be fixed now.